ci/linux-changelog.ps1

```diff
@@ -0,0 +1,389 @@
+#!/bin/env pwsh
+
+# This script contains functions for generating Linux package changelogs from a CHANGELOG.md file. `New-Changelog` generates upstream and packaging changelogs for Debian and RPM-based systems.
+#
+# Upstream changelogs are derived from the root CHANGELOG.md.
+# Change notes for all products are included in each upstream changelog.
+#
+# Packaging changelogs are derived from package/Linux/gateway/CHANGELOG.md for
+# Gateway and package/AgentLinux/CHANGELOG.md for Agent.
+#
+# ## Debian
+# The upstream changelog is included as changelog.gz.
+# The packaging changelog is included as changelog.Debian.gz.
+#
+# ## RPM-based systems
+#
+# The upstream changelog is included as ChangeLog.
+# The packaging changelog is embedded in the spec file.
+#
+# The date used in the changelog is the date of the release in CHANGELOG.md.
+
+
+function Format-EntryLine {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]
+        # The input string to format.
```

Comment:
nitpick: I believe documentation string would come before the annotations typically.